### DISCUSSION TITLE
Addons can not install

### DISCUSSION THREAD

**Discussion by ghost** [dialogic-godot/dialogic#1602] [Q&A]
I downloaded the zip and put the dialogic-main folder in the addons folder. But something is wrong. 

This is part of the error code.
https://github.com/coppolaemilio/dialogic/assets/136042747/be8f5114-0c4f-48ae-a6a9-84a86cbc0313

I'm first time downloading addon. Am I doing somethings wrong?
I don't know how to upload the folder correctly. 
I want to use this addon. Please help me~

**Comment by ghost** [comment]
I succeeded in installing it, but I can't see anything. I created a timeline, but I can't edit any of it. Even if created a character, I cannot select each character. I have no idea how to use this editor even if I read the manual... 

This is all I can see.
<a href="https://ibb.co/3kP3QNX"><img src="https://i.ibb.co/pnk75zD/20230609123159-1.jpg" alt="20230609123159-1" border="0" /></a>
<a href="https://ibb.co/mJySFCK"><img src="https://i.ibb.co/y0kVBnC/20230609123352-1.jpg" alt="20230609123352-1" border="0" /></a>

Is the screen I'm looking at right? How can I use addons?

**Comment by Jowan-Spooner** [comment]
Hi sorry for your trouble and the late response. I'm not entirely sure what the problem is here. 
Usually when installing dialogic first lots of errorrs **are expected** to appear. You should then enable the plugin and restart godot. (This is a problem with godot trying to check the scripts even though they rely on a global autoload that is only added when the plugin is enabled). The next time it should open without errors. 

It's hard for me to tell whats causing the problems in the second message. Only thing I can add is that you seem to be using the text timeline editor which has no "selecting" as it's text based. You can switch to the visual editor with the button in the top right.
Also if you created a character it should appear. Sometimes the resource list doesn't get updated instantly, then you can select your character from the file system dock.
